src/main/java/appeng/parts/misc/PartToggleBus.java

```diff
@@ -50,7 +50,7 @@
 
 public class PartToggleBus extends PartBasicState
 {
-	private static final int REDSTONE_FLAG = 4;
+	private static final int REDSTONE_FLAG = 8;
```

Comment:
Is this saved to NBT anywhere?  Any concern with changing this value?

Reply:
They are for description packet